tempe/README.md

```diff
@@ -11,7 +11,8 @@ while the reference used for RNA does not have ALT-contigs as STAR is not ALT-aw
 ## Script Usage Order
 
 * sbatch tempe/create_genome_reference.sh tempe/tempe_resources.ini
-  * bash shared_resource_creation_scripts/create_bwa_genome_index.sh tempe/tempe_resources.ini
+  * bash tempe/create_bwa_genome_index.sh tempe/tempe_resources.ini
```

Comment:
I'd add a comment
# Uses custom creation script to ensure the genome specific alt file is downloaded so bwa runs in alt-aware mode

Reply:
Updated in [1b8630f](https://github.com/tgen/jetstream_resources/pull/72/commits/1b8630f8e9d810816400fa62860ac052829321ec)